cloudinit/osys/util.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2015 Cloudbase Solutions Srl
+# Copyright 2015 Canonical Ltd.
+# License TBD
+
+__all__ = ('abstractclassmethod', )
+
+
+class abstractclassmethod(classmethod):
```

Comment:
This almost seems like it should be in a file @ ``cloudinit/util.py`` since this seems more generic and useful outside of 'osys' right?